apps/src/applab/redux/screens.js

```diff
@@ -1,10 +1,29 @@
 import Immutable from 'immutable';
+import {
+  sources as sourcesApi,
+  channels as channelsApi,
+} from '../../clientApi';
 
 const CHANGE_SCREEN = 'screens/CHANGE_SCREEN';
 const TOGGLE_IMPORT_SCREEN = 'screens/TOGGLE_IMPORT_SCREEN';
+const IMPORT = {
+  IMPORT_PROJECT: {
+    START_FETCHING: 'screens/importProject/START_FETCHING',
+    FAILED_FETCHING: 'screens/importProject/FAILED_FETCHING',
+    FINISHED_FETCHING: 'screens/importProject/FINISHED_FETCHING',
+  },
+};
+
+var ImportProjectState = Immutable.Record({
```

Comment:
It's questionable. Here's approximately the different set of actions I think I will need:

* IMPORT
  * IMPORT_PROJECT
    * START_FETCHING
    * FINISHED_FETCHING
    * FAILED_FETCHING
  * SELECTION
    * TOGGLE_SCREEN_SELECTION
    * TOGGLE_ASSET_SELECTION
  * COPY
    * START_COPYING_ASSET
    * FINISHED_COPYING_ASSET
    * FAILED_COPYING_ASSET
    * COPY_SCREENS

Definitely open to alternate naming scheme.